sdk/cosmos/azure-cosmos/README.md

```diff
@@ -76,6 +76,28 @@ KEY = os.environ['ACCOUNT_KEY']
 client = CosmosClient(URL, credential=KEY)
 ```
 
+### AAD Authentication
+
+You can also authenticate a client utilizing your service principal's AAD credentials and the azure identity package:
+```Python
+from azure.cosmos import CosmosClient
+from azure.identity import ClientSecretCredential
+
+import os
+URL = os.environ['ACCOUNT_URI']
+TENANT_ID = os.environ['TENANT_ID']
+CLIENT_ID = os.environ['CLIENT_ID']
+CLIENT_SECRET = os.environ['CLIENT_SECRET']
+
+AAD_CREDENTIALS = ClientSecretCredential(
+    tenant_id=TENANT_ID,
+    client_id=CLIENT_ID,
+    client_secret=CLIENT_SECRET)
+
+client = CosmosClient(URL, AAD_CREDENTIALS)
```

Comment:
I'd just make these lowercase variables - looks like we're passing in constants.